src/resources/generated/notification_event_data.rs

```diff
@@ -12,3 +12,7 @@ pub struct NotificationEventData {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub previous_attributes: Option<NotificationEventDataPreviousAttributes>,
 }
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct NotificationEventDataPreviousAttributes {
+}
```

Comment:
This is the only diff to the generated code - trying to figure out where it comes from (but this file is dead code anyway, so does not change anything compilation-wise)